integration-tests/flyway/src/test/java/io/quarkus/it/flyway/FlywayFunctionalityTest.java

```diff
@@ -17,4 +17,5 @@
     public void testFlywayQuarkusFunctionality() {
         when().get("/flyway/migrate").then().body(is("1.0.1"));
     }
+
```

Comment:
Is this line break necessary?

Reply:
I'm using [Quarkus Code Style](https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md) and before Commit, I run "mvn clean install" comands to format the code corretly